packages/backend/src/analytics/LightdashAnalytics.ts

```diff
@@ -56,9 +56,9 @@ type IdentityLinkedEvent = BaseTrack & {
 };
 
 type CreateUserEvent = BaseTrack & {
-    event: 'user.created';
+    event: 'user.created' | 'user.completed';
```

Comment:
I think just a `user.created` event is fine? I'm not sure what the difference between these two events is 🤔

Reply:
@TuringLovesDeathMetal This is a related to the chat we had this week on the new onboarding flow. The user sets his credentials or google auth  ( user.create event ) and then inside the app types the Org name, job title and tracking preferences ( user.completed ). 

@owlas 
I intend to move the connect a project form to be inside the app but until then the project.created event would be triggered before a user.created.
To avoid that,  we could make this PR bigger and I can merge everything at the same time or hold a release until that work is also merged.

Reply:
If we merge this PR `https://github.com/lightdash/lightdash/pull/1211` that moves the project creation inside the app, we can remove the user.completed event and only send the user.created event when the user names the org and sets the tracking values. The same would happen for the org events. We would only trigger the organisation.created event at this point.

Reply:
yep, that makes sense. 

I forgot about the job title, tracking preferences, etc. 

What would be _**truly awesome**_ is if the `user_updated` and `user_created` events had context values for the user (e.g. name, email, tracking, job preferences, etc.) and just use this event for _**all**_ user updates. 

And, we did the same thing for the `organization_updated` and `organization_created` events.

Then have a field that says `update reason` or something which has the update reason e.g. `completed user creation` or `updated email` 

^If that's not possible, then the second-best option for me is:
- sending a `user_created` + `organization_created` event when the user sets up their google auth credentials
- triggering a `user_updated` + `organization_updated` event when they complete their profile would be great 👌 (adding the `update reason` into `user_updated` would be the cherry on top 😄 )

Reply:
The `updated` events _**have**_ to have the context fields that are updated when they "complete" signup though (otherwise, we'll just lose that information).